R/db.R

```diff
@@ -131,6 +131,7 @@ report_db_cols <- function() {
     hash_orderly = "TEXT",
     hash_input = "TEXT",
     hash_resources = "TEXT", # should be json (dict)
+    size_resources = "TEXT", # should be json (dict)
```

Comment:
The db.R file is legacy and all changes will need to go in db2.R and into inst/database/schema.yml.  Can you back these changes out for now please and we'll do that on a separate ticket